recipes/ffmpeg/all/conanfile.py

```diff
@@ -0,0 +1,352 @@
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "postproc": [True, False],
+               "zlib": [True, False],
+               "bzlib": [True, False],
+               "lzma": [True, False],
+               "iconv": [True, False],
+               "freetype": [True, False],
+               "openjpeg": [True, False],
+               "openh264": [True, False],
+               "opus": [True, False],
+               "vorbis": [True, False],
+               "zmq": [True, False],
+               "sdl2": [True, False],
+               "x264": [True, False],
+               "x265": [True, False],
+               "vpx": [True, False],
+               "mp3lame": [True, False],
+               "fdk_aac": [True, False],
+               "webp": [True, False],
+               "ssl": [False, "openssl", "securetransport"],
+               "alsa": [True, False],
+               "pulse": [True, False],
+               "vaapi": [True, False],
+               "vdpau": [True, False],
+               "xcb": [True, False],
+               "appkit": [True, False],
+               "avfoundation": [True, False],
+               "coreimage": [True, False],
+               "audiotoolbox": [True, False],
+               "videotoolbox": [True, False],
+               "qsv": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "postproc": True,
+                       "zlib": True,
+                       "bzlib": True,
+                       "lzma": True,
+                       "iconv": True,
+                       "freetype": True,
+                       "openjpeg": True,
+                       "openh264": True,
+                       "opus": True,
+                       "vorbis": True,
+                       "zmq": False,
+                       "sdl2": False,
+                       "x264": True,
+                       "x265": True,
+                       "vpx": True,
+                       "mp3lame": True,
+                       "fdk_aac": True,
+                       "webp": True,
+                       "ssl": "openssl",
+                       "alsa": True,
+                       "pulse": True,
+                       "vaapi": True,
+                       "vdpau": True,
+                       "xcb": True,
+                       "appkit": True,
+                       "avfoundation": True,
+                       "coreimage": True,
+                       "audiotoolbox": True,
+                       "videotoolbox": True,
+                       "qsv": True}
+    generators = "pkg_config"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os == "Windows" and self.settings.compiler == "gcc" and os.name == "nt"
```

Comment:
`os.name` is a build system property.
```suggestion
        return self.settings.os == "Windows" and self.settings.compiler == "gcc"
```